uaclient/util.py

```diff
@@ -410,10 +410,16 @@ def prompt_choices(msg: str = "", valid_choices: "List[str]" = []) -> str:
     :return: Valid response character chosen.
     """
     value = ""
-    if msg[-1] != " ":
-        msg += " "
-    while value not in valid_choices:
-        value = input(msg).lower().strip()
+    while True:
+        print(msg)
+        value = input("> ").lower()
+        if value in valid_choices:
+            break
+        print(
+            "{} is not one of: {}".format(
+                value, ", ".join([choice.upper() for choice in valid_choices])
```

Comment:
Even though we only have three choices, maybe it is better to create that list outside the while loop to avoid creating one each time the user types something wrong

Reply:
+1 done.